package.json

```diff
@@ -67,7 +67,7 @@
   },
   "dependencies": {
     "@govuk-one-login/di-ipv-cri-common-express": "6.1.0",
-    "@govuk-one-login/one-login-analytics": "^0.0.9",
+    "@govuk-one-login/one-login-analytics": "^1.0.2",
```

Comment:
I think we need to use the latest version of analytics package:
https://www.npmjs.com/package/@govuk-one-login/frontend-analytics

cc @di-aholme @nickhealGDS